common/views/components/Tasl/Tasl.tsx

```diff
@@ -90,7 +90,7 @@ const Title: FunctionComponent<TitleProps> = ({
   if (sourceLink) {
     return (
       <>
-        <a href={sourceLink} property="dc:title" rel="cc:attributionURL">
```

Comment:
So `property` is only accepted on `meta`, and I couldn't find any other `dc:` strings anywhere. I'm removing them but am conscious it might've been put there at the time for a reason and we might need to tweak this information instead of just removing it?

Reply:
The `dc:` prefix stands for Dublin Core, which is something to do with linked data and metadata standards. I don't really know the details tbh; it was something we've only ever followed somewhat half-heartedly and removing it is probably fine.